### PR TITLE
add missing function call to `normalize-catchall-routes` test case

### PR DESCRIPTION
**Pull request by wiiiimm**
### Fixed a Test Case

- Tests fixed: `packages/next/src/build/normalize-catchall-routes.test.ts`, the function call `normalizeCatchAllRoutes(appPaths)` was removed by accident in this [merged commit ](https://github.com/vercel/next.js/commit/efebba80a7ca06b0fdcd619563f13945eb22508a#diff-e3c0d3c6e15d8b035a3252a1ada2b04d5dd33010885fed8f6d689f75aa80bff5) rendering the test case pointless

REF NEXT-1984

### CODE REVIEW

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 9cb3f80c351518d6b4473225cc3550f373a9c679

`pnpm test-dev test/development/app-hmr/hmr.test.ts`
- app-dir-hmr > filesystem changes > should not break when renaming a folder

<details>
<summary>Expand output</summary>

  ● app-dir-hmr › filesystem changes › should not break when renaming a folder

    TIMED OUT: success

    undefined

    Error: elementHandle.innerText: Element is not attached to the DOM

      636 |
      637 |   if (hardError) {
    > 638 |     throw new Error('TIMED OUT: ' + regex + '\n\n' + content + '\n\n' + lastErr)
          |           ^
      639 |   }
      640 |   return false
      641 | }

      at check (lib/next-test-utils.ts:638:11)
      at Object.<anonymous> (development/app-hmr/hmr.test.ts:44:11)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

File: packages/next/src/build/normalize-catchall-routes.test.ts [hunk 1]
```diff
@@ -114,6 +114,7 @@ describe('normalizeCatchallRoutes', () => {
     }
 
     const initialAppPaths = JSON.parse(JSON.stringify(appPaths))
+    normalizeCatchAllRoutes(appPaths)
```
Comment (ijjk) [inline packages/next/src/build/normalize-catchall-routes.test.ts:117]:
```suggestion
    // ensure values are correct after normalizing
    normalizeCatchAllRoutes(appPaths)
```
Let's add a comment so it's more clear this is relevant.

Comment (williamli) [reply packages/next/src/build/normalize-catchall-routes.test.ts:125]: Actually, this function does the normalizing and it's the function after that does the check.

```
    // normalize appPaths against catchAlls
    normalizeCatchAllRoutes(appPaths)

    // ensure values are correct after normalizing
    ....
```

Comment (williamli) [reply packages/next/src/build/normalize-catchall-routes.test.ts:125]: @ijjk I have pushed https://github.com/vercel/next.js/pull/60777/commits/e9ef5b402f210fae652e0a613cf200a748b75f29 with

- revised comment (see above)
- added it to _all_ test cases within the file, not just the one that was restored

**Review by ijjk — APPROVED** [review]

**Pull request merged**